tests/index-tests.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-const resolvePackagePathSync = require('./');
-const Project = require('fixturify-project');
-const fs = require('fs-extra');
-const FIXTURE_ROOT = `${__dirname}/tmp/fixtures/`
-const expect = require('chai').expect;
+var resolvePackagePathSync = require('../');
```

Comment:
Does this require 'Sync' on the end, or not? It *is* a sync routine, but there's no async version.